### DISCUSSION TITLE
Which Issue Templates we need to enable for this sig-release repository?

### DISCUSSION THREAD

**Discussion by FaGru3n** [eclipse-tractusx/sig-release#2] [Ideas]
Hi all,

we are planning issues templates for this repositories.

which labels and information do we need to create within theese templates?

**Comment by efficient0482** [comment]
- suggest a new kit
  - create a poll for each suggestion ?
- request promotion of a kit into the next stage

edit: typo

**Reply by efficient0482** [reply]
additionally something like a "general-issue"

**Comment by FaGru3n** [comment]
https://github.com/eclipse-tractusx/community/pull/10
created firt pull request for first status for the issue-templates

**Comment by FaGru3n** [comment]
Issue Templates at the moment `Feature` and `Bug`
https://github.com/eclipse-tractusx/sig-release/tree/main/.github/ISSUE_TEMPLATE

**Comment by FaGru3n** [comment]
hi @LuisRickert will close this discussion because new ideas will appear after presenting it in pre-alignment meeting 😉
